src/chart/parameter/ParameterSelectionChart.tsx

```diff
@@ -97,6 +100,7 @@ export const NeoParameterSelectionChart = (props: ChartProps) => {
         settings={props.settings}
         allParameters={allParameters}
         compatibilityMode={compatibilityMode}
```

Comment:
Selectors also seem to break when:
- first create a parameter `neodash_person_name` which is a date
- then change type to node property
- the component gets confused about the data type and crashes

Reply:
This one seems resolved.